### DISCUSSION TITLE
Domain is pending verification

### DISCUSSION THREAD

**Discussion by AppitectDigital** [vercel/community#717] [Help] [answered]
Hey,

I have a domain that was previously hooked up to Gitbook (now removed) and when I try to use it on Vercel, I get please verify.

How can I fix this?

**Comment by swarnava** [comment, marked as answer]
Please raise a case using https://vercel.com/support/request with details so that our team can take a look.

**Reply by AppitectDigital** [reply]
Thanks - have submitted :)
